mindgraph/graph.py

```diff
@@ -4,109 +4,109 @@
 from yaml import dump, load
 
 
-class Node(object):
-    """node class"""
+class Task(object):
+    """Task class"""
 
-    def __init__(self, name: str = '', weight: int = 1) -> None:
-        self._dependencies = list()  # type: List[Node]
-        self._threads = list()  # type: List[Node]
+    def __init__(self, name: str = '', priority: int = 1) -> None:
+        self._blockers = list()  # type: List[Task]
+        self._subtasks = list()  # type: List[Task]
         self._name = ''  # type: str
-        self._weight = weight  # type: int
+        self._priority = priority  # type: int
         if type(name) is str:
             self._name = name
         else:
             raise TypeError
 
-    def append(self, newnode) -> "Node":
-        """ Creates a new Node and appends it to threads """
-        if type(newnode) is str:
-            newnode = Node(newnode)
-        elif type(newnode) is not Node:
+    def append(self, newTask) -> "Task":
```

Comment:
this looks like a find/replace error. you probably don't want to introduce camelCase. I suggest `newtask` or `new_task`

Reply:
fixed in 6cae8b6!